spec/acceptance/zabbix_host_spec.rb

```diff
@@ -132,8 +132,40 @@ class { 'zabbix':
         end
       end
 
+      pp4 = <<-EOS
+        zabbix_host { 'test5.example.com':
+          ipaddress   => '127.0.0.5',
+          use_ip      => false,
+          port        => 1051,
+          groups      => ['Virtual machines'],
+          templates   => #{template},
+          macros      => [],
+          tls_accept  => 'cert',
+          tls_connect => 'cert',
+          tls_issuer  => 'Zabbix.com',
+          tls_subject => 'MyClientCertificate',
+        }
+      EOS
+
+      it 'creates host with tls connections configured without errors' do
+        apply_manifest(pp4, catch_failures: true)
+      end
+
+      it 'creates host with tls connections configured without changes' do
+        apply_manifest(pp4, catch_changes: true)
+      end
```

Comment:
Although every other example is like that, we now have [this](https://github.com/voxpupuli/voxpupuli-acceptance#an-idempotent-resource) which make it little bit shorter than running `apply_manifest` 2 times

```suggestion
      it_behaves_like 'an idempotent resource' do
        let(:manifest) do
          <<-EOS
          zabbix_host { 'test5.example.com':
            ipaddress   => '127.0.0.5',
            use_ip      => false,
            port        => 1051,
            groups      => ['Virtual machines'],
            templates   => #{template},
            macros      => [],
            tls_accept  => 'cert',
            tls_connect => 'cert',
            tls_issuer  => 'Zabbix.com',
            tls_subject => 'MyClientCertificate',
          }
        EOS
```